corehq/apps/app_manager/tests/test_suite_session_endpoints.py

```diff
@@ -315,3 +316,135 @@ def test_shadow_module(self):
         )
 
         del self.factory.app.modules[0]
+
+
+@patch_validate_xform()
+@patch_get_xform_resource_overrides()
+@flag_enabled('SESSION_ENDPOINTS')
+class SessionEndpointTestsAdvanced(SimpleTestCase, TestXmlMixin):
+    file_path = ('data',)
+
+    def setUp(self):
+        self.domain = 'test-domain'
+        self.factory = AppFactory(build_version='2.51.0', domain=self.domain)
+        self.parent_case_type = 'mother'
+        self.module, self.form = self.factory.new_advanced_module('advanced', self.parent_case_type)
+
+        builder = XFormBuilder(self.form.name)
+        builder.new_question(name='name', label='Name')
+        self.form.source = builder.tostring(pretty_print=True).decode('utf-8')
+
+    def test_without_computed(self):
```

Comment:
@shubham1g5 Hey Shubham, I have added two tests [with](https://github.com/dimagi/commcare-hq/pull/30707/files#diff-5e51d180dd0fde10d5decd2e10a001042c880b1bdace14e217a7e821a5e11f5bR393) and without the computed datums to test the resulting suite xml. Can you verify that the endpoint xml is as you specified.

Reply:
looks good to me, though is it possible to deploy this on staging for some testing.

Reply:
This should be on staging now. Let me know when you get a chance to test.

Reply:
@orangejenny There seems to be an issue how this PR is interacting with the `remote-request` work you did earlier. I am testing this on stagaing and in the [suite file](https://staging.commcarehq.org/a/shubhamgoyaltest/apps/download/9261779eff8846d485e421dda7861b84/suite.xml), the `sms_form` endpoint is adding up a `command` in stack `claim_command.sms_form.datum_test` definition for which is not present in the suite file. Probably we should not treat the computed datums as case ids here ?

Reply:
Yeah, that sounds likely. @sravfeyn you'll want to ignore computed datums in two places: [here](https://github.com/dimagi/commcare-hq/blob/178f5c43df87e329595b13e0d867e807b0f015d9/corehq/apps/app_manager/suite_xml/post_process/endpoints.py#L46-L49) is where the claim frames are added to the endpoint stack, and [here](https://github.com/dimagi/commcare-hq/blob/178f5c43df87e329595b13e0d867e807b0f015d9/corehq/apps/app_manager/suite_xml/post_process/remote_requests.py#L336-L339) is where the remote-requests for those claim are created.

Is this going to be smoke tested on formplayer / web apps, too?

Reply:
> Is this going to be smoke tested on formplayer / web apps, too?

Do you mean testing for any regressions because of this ? As far as this computed datum feature is concerned, I am only planning to test this out on mobile which should be able to confirm these changes are correct and results into right config.

Reply:
https://github.com/dimagi/commcare-hq/pull/30707/commits/26f419ea9e175a25e8d6c21d2d3b0e1d44ec0236

Reply:
just regressions

Reply:
Offlined with @orangejenny and decided that it should be fine to not do any regression testing since no one is actively using any of session endpoint stuff currently both on mobile and Formplayer.